packages/@sanity/cli/src/commands/debug/printDebugInfo.js

```diff
@@ -120,7 +121,7 @@ function getGlobalConfigLocation() {
   return path.join(configDir, 'sanity', 'config.json')
 }
 
-function gatherGlobalConfigInfo(context) {
+function gatherGlobalConfigInfo() {
```

Comment:
there's an invocation of `gatherGlobalConfigInfo` on ln 98 which doesn't need to get passed the context anymore